rs/tests/system_tests.bzl

```diff
@@ -159,17 +161,18 @@ def system_test(
     """
 
     # Names are used as part of domain names; thus, limit their length
-    if len(name) > 50:
+    if len(name) > 50 or len(variant) > 25:
```

Comment:
Can you split this into two checks with corresponding error messages, or update the message?

Reply:
Actually the most accurate check is:
```python
if len(name + variant_suffix) + 14 > 63:
```
but that's only relevant for tests that [create_dns_records](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/driver/src/driver/farm.rs?L264). So I think it's best if we just drop this check.